subsys/fs/nvs/nvs.c

```diff
@@ -485,7 +486,18 @@ static int nvs_sector_close(struct nvs_fs *fs)
 	return 0;
 }
 
+static int nvs_add_gc_done_ate(struct nvs_fs *fs)
+{
+	struct nvs_ate gc_done_ate;
+
+	LOG_DBG("Adding gc done ate at %x", fs->ate_wra & ADDR_OFFS_MASK);
+	gc_done_ate.id = 0xffff;
+	gc_done_ate.len = 0U;
+	gc_done_ate.offset = (uint16_t)(fs->data_wra & ADDR_OFFS_MASK);
```

Comment:
gc_done_ate.offset: doses it have any future sense to set this value to data_wra?
Not a problem, I'm just curious.

Reply:
No real reason, if it would ever be needed to evaluate where the data stopped after gc it is stored.